soccer/src/soccer/soccerMom/soccerMom.cpp

```diff
@@ -0,0 +1,28 @@
+#include <rclcpp/rclcpp.hpp>
+#include "soccerMom.hpp"
+#include <string>
+#include "std_msgs/msg/string.hpp"
+
+
+namespace tutorial {
+    SoccerMom::SoccerMom() : Node("soccer_mom") {
+        team_color_sub_ = create_subscription<rj_msgs::msg::TeamColor>(
+        referee::topics::kTeamColorTopic, rclcpp::QoS(1).transient_local(),
```

Comment:
`transient_local` QoS is okay here, but if you aren't sure what that means it's worth googling, because we only want it sometimes.